src/components/FeatureInformation/FeatureInformation.scss

```diff
@@ -25,7 +25,17 @@
   }
 
   .wkp-feature-information-body {
-    padding: 10px;
+    & > div {
+      padding: 10px;
+    }
+
+    & > div:first-child {
```

Comment:
@gevos please make the style definitions more explicit. Styling `div`'s without classes makes extending and changing styles very difficult.

Reply:
@mhaertwig I cannot add class names as I do not know them.

There are two cases:

**Without pagination**, the last child is the custom topic popup where the className is always different, e.g. `wkp-infofpw-popup` or another class in another topic:
```
& > div:last-child {
  padding-bottom: 10px;
}
```

**With pagination**, the last child is the pagination  `wkp-pagination-wrapper`:
```
& > div:first-child { 
  padding-bottom: 0;
}

& > div:last-child {
  padding-bottom: 10px;
}
```


I could write this instead, but the first general line would remain:

```
& > div:last-child,
& > div.wkp-pagination-wrapper:last-child {
  padding-bottom: 10px;
}
```

Reply:
To solve this, I have added a className property to each popup.